crates/stdarch-gen/neon.spec

```diff
@@ -1544,6 +1544,28 @@ validate 9, 30, 11, 20, 13, 18, 15, 48
 aarch64 = pmull
 generate poly8x16_t:poly8x16_t:poly16x8_t
 
+/// Floating-point fused Multiply-Add to accumulator(vector)
+name = vfma
+a = 2.0, 3.0, 4.0, 5.0
+b = 6.0, 4.0, 7.0, 8.0
+c = 8.0, 18.0, 12.0, 10.0
+validate 20.0, 30.0, 40.0, 50.0
+
+aarch64 = fma
```

Comment:
This should be `fmadd`?

Reply:
> This should be `fmadd`?

Hi, thanks for reviewing. the way I judge at present is mainly to look at the instruction name, and then according to the result of the disassembly check, fmadd can also pass the check. Is there any other verification method?

Reply:
You can usually find the instruction here: https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics

It only shows the AArch64 instruction though.